<launch>
    <node name="navigation_goals" pkg="mopad_navigation" type="mopad_navigation" output="screen">
    </node>
</launch>
